packages/jellyfish-api-core/__tests__/category/account.test.ts

```diff
@@ -516,4 +516,60 @@ describe('masternode', () => {
       expect(data.length).toBe(64)
     })
   })
+
+  describe('accountHistoryCount', () => {
+    it('should get accountHistoryCount', async () => {
+      await waitForExpect(async () => {
+        const count = await client.account.accountHistoryCount()
+
+        expect(typeof count).toBe('number')
+        expect(count).toBeGreaterThan(0)
+      })
+    })
+
+    it('should get accountHistoryCount with owner as all', async () => {
+      await waitForExpect(async () => {
+        const count = await client.account.accountHistoryCount('all')
+
+        expect(typeof count).toBe('number')
+        expect(count).toBeGreaterThan(0)
```

Comment:
I feel like this should be toBeGreaterThanOrEqual. Don't change first. Let see what others think.

Reply:
Yeah, I am also curious why it did not fail. Likely because the MasterNode container create a MasterNode and that is an account history. Causing it to not always be 0. @canonbrother you happen to know the reason?

@siradji what is the actual number, can we do a deterministic assertion for this?

Reply:
`because the MasterNode container create a MasterNode and that is an account history`
nope, its because of the `waitForWalletCoinbaseMaturity()`

Reply:
@fuxingloh, i am getting 136.

Reply:
Anyways, i have changed the assertion to `toBeGreaterThanOrEqual`.